backend/src/services/dbService.ts

```diff
@@ -136,14 +141,109 @@ FOREIGN KEY (pb_id) REFERENCES ProjectionBundle(id));`,
     }
   }
 
-  async getCurTime(): Promise<Date> {
+  async getInsertUserID(user: User): Promise<number> {
```

Comment:
nit: Zweimal selber Funktionsname sollte NIE sein.

Reply:
Fixed in commit: 6662bb1c0a3996348032e9a5fcd0588fa5273abe